src/app.js

```diff
@@ -1,8 +1,24 @@
 console.info("I'm running...")
 
+
+console.log("---> HW1 - NodeJS modules. NPM");
 import config from "./config/config.json" 
-console.log(config.name);
+console.log(`Logging the name of application (from JSON config file): ${config.name}`);
 
 import { User, Product } from "./models/models"
+console.log("Creating instances of User and Product:")
 new User();
 new Product();
+
+
+console.log("---> HW2 - Async development");
+console.log("Watching for changes in 'data' directory and import CSV files as json:");
+import DirWatcher from "./csv/dirwatcher"
```

Comment:
Usually in Java I created a single responsible modules/classes and not a bunch of homeworks =) Here I divided different homeworks to a kind of groups, every group starts with console.log("---> HWX - description"); and contains all the stuff needed for single homework in one place. May be it would be better to not mix HWs, but extract them into separate modules? e.g. create hw1.js, hw2.js or some kind of this.